examples/connext_dds/lbediscovery/c++11/README.md

```diff
@@ -0,0 +1,142 @@
+if you generated a Visual Studio solution, open the solution and follow the
+regular build process.
+
+## Running the Example
+
+Open two separate command prompt windows for the publisher and subscriber. Run
+the following commands from the build directory (this is necessary to ensure
+the application loads the QoS defined in *USER_QOS_PROFILES.xml*):
+
+On *Windows* systems run:
+
+```sh
+ShapeType_publisher.exe  -d <domain_id> -s <samples_to_send> -v <verbosity 0-3>
+ShapeType_subscriber.exe -d <domain_id> -s <samples_to_receive> -v <verbosity 0-3>
+```
+
+On *UNIX* systems:
+
+```sh
+./ShapeType_publisher  -d <domain_id> -s <samples_to_send> -v <verbosity 0-3>
+./ShapeType_subscriber -d <domain_id> -s <samples_to_receive> -v <verbosity 0-3>
+```
+
+The applications accept up to three arguments:
+
+1. The `<domain_id>`. Both applications must use the same domain ID in order to
+    communicate. The default is 0. If you plan to use a `domain_id` different
+    than 0, you should change the value of the `domain_id` attribute of the
+    `<domain>` tag in the USER_QOS_PROFILES.xml.
+
+2. How long the examples should run, measured in samples. The default is
+    infinite.
+
+3. The verbosity of the application.
+
+## Customizing the Build
+
+### Configuring Build Type and Generator
+
+By default, CMake will generate build files using the most common generator for
+your host platform (e.g., Makefiles on Unix-like systems and Visual Studio
+solution on Windows), \. You can use the following CMake variables to modify the
+default behavior:
+
+- `-DCMAKE_BUILD_TYPE` -- specifies the build mode. Valid values are Release
+    and Debug. See the [CMake documentation for more details.
+    (Optional)](https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html)
+
+- `-DBUILD_SHARED_LIBS` -- specifies the link mode. Valid values are ON for
+    dynamic linking and OFF for static linking. See [CMake documentation for
+    more details.
+    (Optional)](https://cmake.org/cmake/help/latest/variable/BUILD_SHARED_LIBS.html).
+    **Note: LBED can only be linked dynamically.**
+
+- `-G` -- CMake generator. The generator is the native build system to use
+    build the source code. All the valid values are described in the
+    CMake documentation [CMake Generators
+    Section.](https://cmake.org/cmake/help/latest/manual/cmake-generators.7.html)
+
+For example, to build an example in Debug/Dynamic mode run CMake as follows:
+
+```sh
+cmake -DCMAKE_BUILD_TYPE=Debug -DBUILD_SHARED_LIBS=ON .. \
+    -G "Visual Studio 15 2017" -A x64
+```
+
+### Configuring Connext DDS Installation Path and Architecture
+
+The CMake build infrastructure will try to guess the location of your Connext
+DDS installation and the Connext DDS architecture based on the default settings
+for your host platform. If you installed Connext DDS in a custom location, you
+can use the CONNEXTDDS_DIR variable to indicate the path to your RTI Connext DDS
+installation folder. For example:
```

Comment:
Change all instances of Connext DDS in this paragraph to Connext. (We are phasing out the "DDS" in "Connext DDS.")